### ISSUE TITLE
Score a task reporting failure even though ID is correct

### ISSUE THREAD

**Issue by citrusella** [goldfndr/tasker-habitrpg#6]
Task scoring is coming up with a message about the scoring failing, even though I have repeatedly checked the ID and verified it to be correct. Further, Habitica itself is granting me the relevant changes (task value, gold, XP). So... as far as I can tell, the process is thinking the ID is wrong whether it is or not?

**Comment by thewub** [comment]
Having the same issue. I checked with the Habitica User Data Display Tool, and the task has definitely been scored correctly, even though I get the "Failed to score task ... doublecheck its ID" message.

**Comment by 0xCyberFortress** [comment]
@goldfndr I'm afraid even after that fix in the newest version of the script, I'm now getting the same error as above. I've checked the script, and have confirmed the changes you made in your fix earlier. For some reason it's still giving me the same error though. I've confirmed my taskid's are indeed correct, and also verified my API creds. Everything is 100% right, so this has to be a problem on your end :(
